docs/_coverpage.md

```diff
@@ -1,13 +0,0 @@
-<p align="center"><img src="https://s3.amazonaws.com/dydx-assets/dydx_logo_white.svg" width="256" /></p>
```

Comment:
I like the coverpage :( can we keep it?

We can add links to all the different projects on the coverpage which would be a good way to get to them

Reply:
There is still a homepage, the coverpage is just gratuitous and adds an extra click.

Reply:
If there's anything you want to add to homepage, put it in README.md